docker/shadow-hand/kinetic/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM shadowrobot/build-tools:xenial-kinetic-ide
+
+ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/master/ansible/deploy.sh"
+
+ENV PROJECTS_WS=/home/user/projects/shadow_robot
+RUN echo "Running one-liner" && \
+    apt-get update && \
+    wget -O /tmp/oneliner "$( echo "$remote_shell_script" | sed 's/#/%23/g' )" && \
+    chmod 755 /tmp/oneliner && \
+    gosu $MY_USERNAME /tmp/oneliner -w $PROJECTS_WS/base -b "kinetic-devel" -v "kinetic"  && \
+    
+    echo "Installing mongodb" && \
+    cd $PROJECTS_WS/base_deps/src && \
+    
+    # installing mongocxx driver for the warehouse && \
```

Comment:
I am using 
```bash
echo "installing mongocxx driver for the warehouse" && \
```
This will make it a little bit easier to identify place where script has failed